kategory/src/main/kotlin/kategory/instances/OptionInstances.kt

```diff
@@ -81,3 +81,12 @@ interface OptionInstances :
             fa.ev().orElse({ f(Unit).ev() })
 
 }
+
+/**
+ * Dummy SemigroupK instance to be able to test laws for SemigroupK.
+ */
+class OptionSemigroupK : SemigroupK<Option.F> {
+    override fun <A> combineK(x: HK<Option.F, A>, y: HK<Option.F, A>): HK<Option.F, A> {
```

Comment:
When implementing typeclasses an IDEA automatically suggests: `HK<Option.F, A>` what the IDE really wants is for you to manually go and change that to `Option<A>` xD. This allows people using the instances not to have to upcast with `ev()` all over the place.